.github/workflows/container-image-buildah.yml

```diff
@@ -0,0 +1,100 @@
+name: Container Image
+
+on:
+  workflow_dispatch: {}
+  workflow_call: {}
+  schedule:
+    # every Wednesday morning
+    - cron: 7 7 * * 3
```

Comment:
Will it keep building the same version again and again if there are is no activity for weeks?

Reply:
Yes, it can be disabled. But I find it useful to build with updated base image potentially fixing security issues.

Reply:
And making sure that image sill builds with latest base image.